src/services/blocks/BlocksService.ts

```diff
@@ -537,7 +534,7 @@ export class BlocksService extends AbstractService {
 
 		// Now that we know the exact runtime supports fee calcs, make sure we have
 		// the weights in the store
```

Comment:
Nit: I think this comment is now out of date :)

Reply:
Yea will get removed/updated. Good catch!